test/test_dynamic_shapes.py

```diff
@@ -414,8 +420,7 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         self.assertTrue(sym_int_encountered)
 
     @skipIfNoSympy
-    @unittest.mock.patch('sys.stdout', new_callable=io.StringIO)
```

Comment:
The mocking here has been annoying me for some time as it impedes the use of debugging prints. Turns out we don't need to mock sys.stdout anyway.